docs/architecture/APIDescription.md

```diff
@@ -0,0 +1,13 @@
+# API
+
+## Endpoints
+
+1. CRUDS (create, read, update, delete, search) for product's group (product_groups)
+
+## Description of entity product_groups
+
+1. Info
+    1. Name
+    2. Description
+    3. Properties
+2. ProductPropertyId - identification groups properties(foreign / local)
```

Comment:
May be it was not completely clear the goals of this component.
Its is implied to serve the group of products like in Yandex Market. For example: "washing machine", "refrigerators", etc.
Each group has its own set of properties. Those properties are offered when a product is edited.
Also those properties are used for filtering